# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../softdog/hostfixes.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./softdog.c~	Wed Jan  6 16:14:23 1999
+++ ./softdog.c	Sat Apr 29 08:28:22 2000
@@ -26,6 +26,12 @@
 #include <signal.h>
 #include <errno.h>
 #include <getopt.h>
+
+#define _LINUX_STRING_H_
+#define __KERNEL__
+#  include <linux/kdev_t.h>
+#  include <linux/list.h>
+#undef __KERNEL__
 #include <linux/fs.h>
 
 #define VERSION "1.21"
